config/database.yml.example

```diff
@@ -18,6 +19,7 @@ development:
   password: 
   encoding: utf8
   database: publiclaboratory
+  strict: false
```

Comment:
Same as above

Reply:
This was added by @jywarren in the update process